test/berthelothercourtessen_rate_vec_unit.C

```diff
@@ -91,13 +93,13 @@ int check_rate_and_derivative(const PairScalars & rate_exact, const PairScalars
     int return_flag(0);
    for (unsigned int tuple=0; tuple != ANTIOCH_N_TUPLES; ++tuple)
    {
-     if( excessive_difference(T[2*tuple],   rate[2*tuple],     rate_exact[2*tuple],     tol, "rate values") )
+     if( excessive_difference<Scalar>(T[2*tuple],   rate[2*tuple],     rate_exact[2*tuple],     tol, "rate values") )
```

Comment:
Weird.  Why was this necessary?  Is the vexcl::vec::operator[] return type not the underlying scalar type?